public/css/login.module.css

```diff
@@ -108,17 +107,17 @@
   margin-block: 50px;
 }
 
-    & button {
-        height: 50px;
-        font-size: 26px;
-        border-radius: 10px;
-        background-color: rgba(255, 255, 255, 0);
-        color: #fff;
-        border: 4px solid #fff;
-        user-select: none;
-        cursor: pointer;
-        transition: 300ms;
-        font-family: poke-font;
+& button {
+  height: 50px;
+  font-size: 26px;
+  border-radius: 10px;
+  background-color: rgba(255, 255, 255, 0);
+  color: #fff;
+  border: 4px solid #fff;
+  user-select: none;
+  cursor: pointer;
+  transition: 300ms;
+  font-family: poke-font;
```

Comment:
no need to specify the font family anymore after the change you made to the general css file (you can see the `& button` font-family is crossed off)
![image_2024-12-24_222826862](https://github.com/user-attachments/assets/32b95019-2232-4b3b-9f11-1a97788e06de)